### PR TITLE
Submission controller

### PR DESCRIPTION
**Pull request by ctlong**
Controller and views for submissions.

### CODE REVIEW

File: app/controllers/submissions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      render json: {updated: true}, status: :success
+    end
+  end
+
+  def delete
+    @submissions = Content.find(params[:content_id]).submissions
+    if @submissions.nil?
+      render json: {message: "Cannot find submissions"}, status: :not_found
+    else
+      @submissions.each do |submission|
```
Comment (h8rry) [inline app/controllers/submissions_controller.rb]: You can do `@submissions.destroy_all`

File: app/controllers/submissions_controller.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
+class SubmissionsController < ApplicationController
+  def index
+    @submissions = Submission.where(params[:submission_key])
+  end
+
+  def create
+    @submission = Content.find(params[:content_id]).submissions.create(submission_params)
```
Comment (h8rry) [inline app/controllers/submissions_controller.rb]: If you are going to go for the new approach, then you don't need an endpoint for submissions for every single content. You should just have ONE endpoint to create ALL answers for ONE submission.

File: app/controllers/submissions_controller.rb [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  end
+
+  def show
+    @submission = Submission.find(params[:id])
+  end
+
+  def update
```
Comment (h8rry) [inline app/controllers/submissions_controller.rb]: Can you modify a submission though? It's a report / survey.

**Comment by h8rry** [comment]
@ctlong Please review

**Comment by ctlong** [comment]
@ggharry Reviewed and modified.

File: app/controllers/submissions_controller.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+  def show
+    @submission = Submission.find(params[:id]).includes(:answers)
+  end
+
+  def delete
+    @submissions = User.find(params[:user_id]).submissions
+    if @submissions.nil?
+      render json: {message: "Cannot find submissions"}, status: :not_found
+    elsif @submissions.destroy_all
```
Comment (h8rry) [inline app/controllers/submissions_controller.rb]: same here as https://github.com/WDI-HK-8/Formidable-web/pull/7#discussion_r39002702

**Comment by h8rry** [comment]
The rest looks good. Remember to rebase after merging #7 and merge.

**Pull request merged**